src/test/CMakeLists.txt

```diff
@@ -489,8 +491,19 @@ if(${TRITON_ENABLE_METRICS})
       GTest::gmock
       prometheus-cpp::core
       protobuf::libprotobuf
+      ${CNMEM_LIBRARY}
+      CUDA::cudart
```

Comment:
Why is the cuda link needed?

Reply:
Pinned memory modules use CUDA functions. To prevent errors during building, the above CUDA linkage has been added.